### ISSUE TITLE
Investigate switching Microsoft.Net.Http reference to System.Net.Http 

### ISSUE THREAD

**Issue by gb92** [Azure/azure-mobile-apps-net-client#82]
Current reference to [Microsoft.Net.Http](http://www.nuget.org/packages/Microsoft.Net.Http/) package has dependencies on Microsoft.Bcl.Build that breaks projects using the new project.json structure. A change to reference [System.Net.Http](http://www.nuget.org/packages/System.Net.Http/4.0.0) would remove the dependency on Bcl and Bcl.Build

**Comment by dhei** [comment]
@adrianhall We will do this as part of moving to .NETStandard, which would be milestone >= 4.0.0.

**Comment by InquisitorJax** [comment]
Is there any eta for the 4.0 release? This issue is preventing .NET standard upgrade on projects.

**Comment by adrianhall** [comment]
No, we don't have a date for the milestone as yet.  We are working on it.

**Comment by InquisitorJax** [comment]
@adrianhall what is the meaning of the transferred label?

**Comment by adrianhall** [comment]
We are consolidating all bugs for all client and server SDKs into our centralized bug tracking in VSTS.  In the future, filing a bug will automatically create a linked bug in VSTS.  However, we needed to do an initial transfer.  Rather than transfer everything, we opted to go through the entire issue list and close those issues which we would never do, and transfer only those that we are likely to work on.  The transferred label indicates we transferred the bug into VSTS (so it's still in play).
